### PR TITLE
Use GitHub Actions

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: ci/publish [hunk 1]
```diff
@@ -30,18 +38,18 @@ if [ ${COMMIT} == TRUE ]
...
     git commit -m "[skip ci] Update the minor version"
-    git push origin ${CIRCLE_BRANCH}
+    git push ${GIT_BRANCH}
 fi
 
 DEPLOY_PIP=FALSE
 MAJOR_VERSION_TAG=${MAJOR_VERSION}
 # Get the version to be published
-if [ "${TAG}" != "" ]
+if [ "${TAG}" ]
```
Comment (danduk82) [inline ci/publish]: `[ -n "${TAG}" ]` ... but ok...

File: ci/test-app [hunk 2]
```diff
@@ -12,35 +12,29 @@ cd ${HOME}/workspace/testgeomapfishapp/
...
-    geoportal/testgeomapfishapp_geoportal/locale/fr/LC_MESSAGES/testgeomapfishapp_geoportal-client.po
-docker-compose exec geoportal update-po de > \
-    geoportal/testgeomapfishapp_geoportal/locale/de/LC_MESSAGES/testgeomapfishapp_geoportal-client.po
-docker-compose exec geoportal theme2fts
+docker-compose exec -T geoportal bash -c 'PGHOST=externaldb PGDATABASE=test wait-db;'
+docker-compose exec -T geoportal wait-db
+docker-compose exec -T geoportal alembic --config=alembic.ini --name=main upgrade head
+docker-compose exec -T geoportal alembic --config=alembic.ini --name=static upgrade head
+docker-compose exec -T geoportal create-demo-theme
+docker-compose exec -T tools update-po $(id --user) $(id --group) en fr de
```
Comment (danduk82) [inline ci/test-app]: and `it`?

File: ci/test-app [hunk 3]
```diff
@@ -12,35 +12,29 @@ cd ${HOME}/workspace/testgeomapfishapp/
...
+ci/run-dc-logs docker-compose exec -T geoportal ci/waitwsgi https://front/themes
 for path in c2c/health_check c2c/health_check?max_level=9 c2c/health_check?checks=check_collector "layers/test/values/type enum" admin/layertree admin/layertree/children
 do
-    ci/run-dc-logs docker-compose exec geoportal ci/test-new-project https://front/${path}
-    #docker-compose exec geoportal curl --insecure https://front/c2c/debug/stacks?secret=c2c
+    ci/run-dc-logs docker-compose exec -T geoportal ci/test-new-project https://front/${path}
+    #docker-compose exec -T geoportal curl --insecure https://front/c2c/debug/stacks?secret=c2c
 done
-ci/run-dc-logs docker-compose exec geoportal ci/test-new-project 'http://qgisserver:8080/mapserv_proxy?SERVICE=WMS&REQUEST=GetCapabilities'
+ci/run-dc-logs docker-compose exec -T geoportal ci/test-new-project 'http://qgisserver:8080/mapserv_proxy?SERVICE=WMS&REQUEST=GetCapabilities'
```
Comment (danduk82) [inline ci/test-app]: don't we need a WMS version in the request? Ok it is optional, but I don't know if we want to be sure if given versions are supported we could explicitely do a couple of tests.

**Comment by sbrunner** [comment]
Done :-)

**Pull request merged**